app/Project.php

```diff
@@ -37,4 +36,23 @@ public function course() {
 	public function team() {
 		return $this->belongsTo('App\Team');
 	}
+
+	// Get the project collaborators
+	public function collaborators() {
+		return $this->team()->first()->members();
```

Comment:
hm esto no regresa los suppliers, sino sólo los team members verdad?

cuál es la diferencia entre llamar esto y llamar $project->team->members directamente?

Reply:
No estoy seguro pero creo que no est&aacute;n asociados los suppliers todav&iacute;a al proyecto, that's not been worked on yet.

Reply:
Estamos trabajando en los invites en integrador, entonces como quiera necesitamos eso. No sé si @osdagoso esté trabajando en los suppliers por separado (?)